src/VisualStudio/Core/Def/LanguageService/AbstractLanguageService`2.VsCodeWindowManager.cs

```diff
@@ -236,6 +236,14 @@ public int RemoveAdornments()
             // that ReleaseOutline will be called on the old window before GetOutline is called for the new window. 
             int IVsDocOutlineProvider.GetOutline(out IntPtr phwnd, out IOleCommandTarget? ppCmdTarget)
             {
+                var enabled = _globalOptions.GetOption(DocumentOutlineOptionsMetadata.EnableDocumentOutline);
```

Comment:
We don't get called if the user never had the document outline window open in the first place, right? This is just a mitigation for those who had it open for other languages and didn't notice?

Reply:
That is correct